README.md

```diff
@@ -257,6 +258,17 @@ json empty_object_explicit = json::object();
 json array_not_object = json::array({ {"currency", "USD"}, {"value", 42.99} });
 ```
 
+### Validation
+
+You can validate a json string using the accept() method:
+
+```cpp
+bool success = json::accept("Invalid json string");
+// success is false
+```
+
+It is highly recommended to call accept() before parse() when receiving json strings from an untrusted source (a web server for example) as parse() can cause fatal errors when passed malformed json.
```

Comment:
I do not share this recommendation. The parse method would throw in case of a parse error, which is perfectly fine. Calling accept before just doubles the processing time.